### PR TITLE
Scala Language support for VDP Sample Code.

### PR DESCRIPTION
**Pull request by AravindaM**
1. Supports both Mutual Authentication and XPayToken based Authentication 
2. Sample Test cases for,

   - VisaDirect : Pushfundstransactions
   - Cybresource : PaymentAuthorization

3. Todo : Adding test cases for the rest of the APIs.

### CODE REVIEW

**Comment by ankeshanand** [comment]
Thanks for the PR AravindaM. Would be helpful if you could add samples for more APIs.

**Comment by AravindaM** [comment]
Thanks @ankeshanand for quickly reviewing and proceeding to merge. Sure i'm planning to add the rest of the test cases as well.

**Comment by ankeshanand** [comment]
@avmohan Can you review this PR?

**Comment by AravindaM** [comment]
Appreciate if someone can review the code.

**Comment by ankeshanand** [comment]
@AravindaM Can we approve this if everything looks okay?

**Comment by avmohan** [comment]
@ankeshanand - Sure.. I don't see any issue

File: vdp-scala/README.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# Scala sample code for Visa API calls
+
+## Prerequisites
+
+### [Scala] (https://www.scala-lang.org/download/install.html)
```
Comment (AravindaM) [inline vdp-scala/README.md]: Fixed. yes it is because of the space.

File: vdp-scala/src/main/scala/libs/ConfgUtils.scala [hunk 2]
```diff
@@ -0,0 +1,16 @@
+package libs
+
+import com.typesafe.config.ConfigFactory
+
+/**
+  * Created by Aravinda on 5/16/2017.
+  */
+object ConfgUtils {
```
Comment (AravindaM) [inline vdp-scala/src/main/scala/libs/ConfgUtils.scala]: Corrected.

**Comment by AravindaM** [comment]
Hi @avmohan Thanks for you time on reviewing the code. @ankeshanand I have done the changes requested.

**Pull request merged**